test/e2e/servicebindingrequest_test.go

```diff
@@ -195,8 +195,13 @@ func assertSBRStatus(
 	}
 
 	for i, condition := range sbr.Status.Conditions {
-		if condition.Type != conditions.BindingReady && condition.Status != corev1.ConditionTrue {
-			return fmt.Errorf("Condition.Type and Condition.Status is '%s' and '%s' instead of '%s' and '%s'", sbr.Status.Conditions[i].Type, sbr.Status.Conditions[i].Status, conditions.BindingReady, corev1.ConditionTrue)
+		if condition.Type != servicebindingrequest.BindingReady && condition.Status != corev1.ConditionTrue {
+			return fmt.Errorf(
+				"Condition.Type and Condition.Status is '%s' and '%s' instead of '%s' and '%s'",
+				sbr.Status.Conditions[i].Type,
+				sbr.Status.Conditions[i].Status,
+				servicebindingrequest.BindingReady,
+				corev1.ConditionTrue)
```

Comment:
@isutton Is these e2e test changes going to work in master? If so, can we make it as a separate PR and merge it first?

Reply:
@baijum this change was made to move conditions symbols to the `servicebindingrequest` package, and to split a really long line.

I believe there isn't any benefit in doing this; the changes performed in that file are basically slightly adjusting the test to add a prefix in etcd tests and adding `require.Eventually` in some places that should've been retried.

There were no changes whatsoever related to the behavior of the test.